
===========================
log level: log_successful_tests; error type: no error;

Entering test case "error_on_demand"
99: info: check 'no error' passed
Leaving test case "error_on_demand"

===========================
log level: log_successful_tests; error type: user message;

Entering test case "error_on_demand"
message
Test case error_on_demand did not check any assertions
Leaving test case "error_on_demand"

===========================
log level: log_successful_tests; error type: user warning;

Entering test case "error_on_demand"
107: warning in "error_on_demand": warning
Test case error_on_demand did not check any assertions
Leaving test case "error_on_demand"

===========================
log level: log_successful_tests; error type: user non-fatal error;

Entering test case "error_on_demand"
111: error in "error_on_demand": non-fatal error
Leaving test case "error_on_demand"

===========================
log level: log_successful_tests; error type: cpp exception;

Entering test case "error_on_demand"
0: fatal error in "error_on_demand": std::runtime_error: test std::runtime error what() message
121: last checkpoint: error_on_demand() throw runtime_error
Leaving test case "error_on_demand"

===========================
log level: log_successful_tests; error type:  system error;

Entering test case "error_on_demand"
115: fatal error in "error_on_demand": fatal error
Leaving test case "error_on_demand"

===========================
log level: log_test_suites; error type: no error;

Entering test case "error_on_demand"
Leaving test case "error_on_demand"

===========================
log level: log_test_suites; error type: user message;

Entering test case "error_on_demand"
message
Test case error_on_demand did not check any assertions
Leaving test case "error_on_demand"

===========================
log level: log_test_suites; error type: user warning;

Entering test case "error_on_demand"
107: warning in "error_on_demand": warning
Test case error_on_demand did not check any assertions
Leaving test case "error_on_demand"

===========================
log level: log_test_suites; error type: user non-fatal error;

Entering test case "error_on_demand"
111: error in "error_on_demand": non-fatal error
Leaving test case "error_on_demand"

===========================
log level: log_test_suites; error type: cpp exception;

Entering test case "error_on_demand"
0: fatal error in "error_on_demand": std::runtime_error: test std::runtime error what() message
121: last checkpoint: error_on_demand() throw runtime_error
Leaving test case "error_on_demand"

===========================
log level: log_test_suites; error type:  system error;

Entering test case "error_on_demand"
115: fatal error in "error_on_demand": fatal error
Leaving test case "error_on_demand"

===========================
log level: log_messages; error type: no error;


===========================
log level: log_messages; error type: user message;

message
Test case error_on_demand did not check any assertions

===========================
log level: log_messages; error type: user warning;

107: warning in "error_on_demand": warning
Test case error_on_demand did not check any assertions

===========================
log level: log_messages; error type: user non-fatal error;

111: error in "error_on_demand": non-fatal error

===========================
log level: log_messages; error type: cpp exception;

0: fatal error in "error_on_demand": std::runtime_error: test std::runtime error what() message
121: last checkpoint: error_on_demand() throw runtime_error

===========================
log level: log_messages; error type:  system error;

115: fatal error in "error_on_demand": fatal error

===========================
log level: log_warnings; error type: no error;


===========================
log level: log_warnings; error type: user message;


===========================
log level: log_warnings; error type: user warning;

107: warning in "error_on_demand": warning

===========================
log level: log_warnings; error type: user non-fatal error;

111: error in "error_on_demand": non-fatal error

===========================
log level: log_warnings; error type: cpp exception;

0: fatal error in "error_on_demand": std::runtime_error: test std::runtime error what() message
121: last checkpoint: error_on_demand() throw runtime_error

===========================
log level: log_warnings; error type:  system error;

115: fatal error in "error_on_demand": fatal error

===========================
log level: log_all_errors; error type: no error;


===========================
log level: log_all_errors; error type: user message;


===========================
log level: log_all_errors; error type: user warning;


===========================
log level: log_all_errors; error type: user non-fatal error;

111: error in "error_on_demand": non-fatal error

===========================
log level: log_all_errors; error type: cpp exception;

0: fatal error in "error_on_demand": std::runtime_error: test std::runtime error what() message
121: last checkpoint: error_on_demand() throw runtime_error

===========================
log level: log_all_errors; error type:  system error;

115: fatal error in "error_on_demand": fatal error

===========================
log level: log_cpp_exception_errors; error type: no error;


===========================
log level: log_cpp_exception_errors; error type: user message;


===========================
log level: log_cpp_exception_errors; error type: user warning;


===========================
log level: log_cpp_exception_errors; error type: user non-fatal error;


===========================
log level: log_cpp_exception_errors; error type: cpp exception;

0: fatal error in "error_on_demand": std::runtime_error: test std::runtime error what() message
121: last checkpoint: error_on_demand() throw runtime_error

===========================
log level: log_cpp_exception_errors; error type:  system error;

115: fatal error in "error_on_demand": fatal error

===========================
log level: log_system_errors; error type: no error;


===========================
log level: log_system_errors; error type: user message;


===========================
log level: log_system_errors; error type: user warning;


===========================
log level: log_system_errors; error type: user non-fatal error;


===========================
log level: log_system_errors; error type: cpp exception;


===========================
log level: log_system_errors; error type:  system error;

115: fatal error in "error_on_demand": fatal error

===========================
log level: log_fatal_errors; error type: no error;


===========================
log level: log_fatal_errors; error type: user message;


===========================
log level: log_fatal_errors; error type: user warning;


===========================
log level: log_fatal_errors; error type: user non-fatal error;


===========================
log level: log_fatal_errors; error type: cpp exception;


===========================
log level: log_fatal_errors; error type:  system error;

115: fatal error in "error_on_demand": fatal error

===========================
log level: log_nothing; error type: no error;


===========================
log level: log_nothing; error type: user message;


===========================
log level: log_nothing; error type: user warning;


===========================
log level: log_nothing; error type: user non-fatal error;


===========================
log level: log_nothing; error type: cpp exception;


===========================
log level: log_nothing; error type:  system error;

